### ISSUE TITLE
target logit and its curves

### ISSUE THREAD

**Issue by xuezhongcailian** [happynear/AMSoftmax#8]
hi,I read your paper and your target logit curves code ,I feel puzzled,your paper say that Wf is also called as the target logit ,but I feel that your  target logit curves is  not correspond with the definied Wf,it seems  not to think about the f.

**Comment by happynear** [comment]
The definition of **logit** is from this paper: https://arxiv.org/pdf/1701.06548.pdf (z_i in Section 3). It originally means the values that are fed into Softmax loss, and we define the **target logit** as the logit which is corresponding to the ground truth label. 

In the original Softmax, the target logit is W_y^T f. In AM-Softmax, we normalize the W and f and modify the target logit to be s*(W_y^T f - m). Here y is the ground-truth label of the optimizing sample.

**Comment by xuezhongcailian** [comment]
hi，I read your target logit curves matlab code on your github draw_psi_curve.m ,I always feel the Angular softmax and AM softmax curves plot the ψ(θ).but conventional softmax plot cos(θ), the target logit is W_y^T f.Angular softmax target logit ||f|| ψ(θ),I feel not  corresponding.

**Comment by happynear** [comment]
You are right. I will change the y-label to ψ(θ) in the final version of my paper.

**Comment by xuezhongcailian** [comment]
I do not  know whether arcface target logit curves have also the problem.

**Comment by happynear** [comment]
Yeah... ArcFace uses the same figure, and suffers from the same problem.
